dax/test/dax/dax_test.go

```diff
@@ -507,6 +514,134 @@ func TestDAXIntegration(t *testing.T) {
+		db, err = mdsClient.DatabaseByName(ctx, orgID, dbname)
+		if assert.Error(t, err) {
+			assert.Contains(t, err.Error(), "database name 'dbname1' does not exist")
+			// TODO(tlt): replace the previous line with the following once we
+			// have threaded error codes through the http calls.
+			// assert.True(t, errors.Is(err, dax.ErrDatabaseNameDoesNotExist))
+		}
+		assert.Nil(t, db)
+
+		tbl1, err = mdsClient.TableByName(ctx, qdbid, dax.TableName(defs.Keyed.Name(0)))
+		if assert.Error(t, err) {
+			assert.Contains(t, err.Error(), "table name 'keyed' does not exist")
+			// TODO(tlt): replace the previous line with the following once we
+			// have threaded error codes through the http calls.
+			// assert.True(t, errors.Is(err, dax.ErrTableNameDoesNotExist))
+		}
+		assert.Nil(t, tbl1)
+
+		tbl2, err = mdsClient.TableByName(ctx, qdbid, dax.TableName(defs.Unkeyed.Name(0)))
+		if assert.Error(t, err) {
+			assert.Contains(t, err.Error(), "table name 'unkeyed' does not exist")
+			// TODO(tlt): replace the previous line with the following once we
+			// have threaded error codes through the http calls.
+			// assert.True(t, errors.Is(err, dax.ErrTableNameDoesNotExist))
+		}
+		assert.Nil(t, tbl2)
+	})
+
+	t.Run("Delete_Table", func(t *testing.T) {
+		mc := test.MustRunManagedCommand(t)
+		defer mc.Close()
+		svcmgr := mc.Manage()
+
+		// Set up MDS client.
+		mdsClient := mdsclient.New(svcmgr.MDS.Address(), svcmgr.Logger)
+
+		// Create database.
+		qdb.Options.WorkersMin = 1
+		qdb.Options.WorkersMax = 1
+		assert.NoError(t, mdsClient.CreateDatabase(context.Background(), qdb))
+
+		testconfigs := basicTableTestConfig(qdbid, defs.Keyed)
+		for i := range testconfigs {
+			testconfigs[i].skipQuery = true
+		}
+		runTableTests(t,
+			svcmgr.Queryer.Address(),
+			testconfigs...,
+		)
+
+		fmt.Printf("%s\n", mc.Config.Computer.Config.DataDir)
+		err := filepath.WalkDir(mc.Config.Computer.Config.DataDir, func(path string, d fs.DirEntry, err error) error {
+			if err != nil {
+				return err
+			}
+			name := strings.TrimPrefix(path, mc.Config.Computer.Config.DataDir)
+			numSlash := strings.Count(name, "/")
+			for i := 0; i < numSlash; i++ {
+				fmt.Print(" ")
+			}
+			fmt.Printf("%s\n", name)
+			return nil
+		})
+		if err != nil {
+			t.Fatalf("walking dir: %v", err)
+		}
+
+		resp := runSQL(t, svcmgr.Queryer.Address(), testconfigs[0].qdbid, "drop table keyed")
+		fmt.Println("RESP: ", resp)
+
+		time.Sleep(time.Second * 5)
+
+		fmt.Printf("%s\n", mc.Config.Computer.Config.DataDir)
+		err = filepath.WalkDir(mc.Config.Computer.Config.DataDir, func(path string, d fs.DirEntry, err error) error {
+			if err != nil {
+				return err
+			}
+			name := strings.TrimPrefix(path, mc.Config.Computer.Config.DataDir)
+			numSlash := strings.Count(name, "/")
+			for i := 0; i < numSlash; i++ {
+				fmt.Print(" ")
+			}
+			fmt.Printf("%s\n", name)
+			return nil
+		})
+		if err != nil {
+			t.Fatalf("walking dir: %v", err)
+		}
```

Comment:
finish this test... not actually checking anything. also need to clean up the duplicated walk func

Reply:
just added some checks to see if the directories are empty or not